### ISSUE TITLE
[2018-09] AbstractWordAwareDoubleClickStrategy does no longer compile

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-eclipse#759]
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 11:20 min
[INFO] Finished at: 2018-07-23T06:43:36+00:00
[INFO] Final Memory: 191M/2073M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:1.2.0:compile (default-compile) on project org.eclipse.xtext.ui: Compilation failure: Compilation failure:
[ERROR] /var/jenkins_home/workspace/xtext-eclipse_master-OX2L6OFRZ75ZC7A3K3BJAQVU3V3QABS2LI4URYKVIRLWPPW5BHSQ/org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/doubleClicking/AbstractWordAwareDoubleClickStrategy.java:[39]
[ERROR] breakIter.setText(new DocumentCharacterIterator(document));
[ERROR] ^^^^^^^
[ERROR] The method setText(CharSequence) is ambiguous for the type BreakIterator
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
```

**Comment by cdietrich** [comment]
ICU 62 has a new 

```
    public void setText(CharSequence newText) {
        setText(new CSCharacterIterator(newText));
    }
```

**Comment by cdietrich** [comment]
Fixed in 2.15
